tools/postgres-init-dev-db

```diff
@@ -24,7 +24,7 @@ if [[ $# == 0 ]]; then
     USERNAME=zulip
     PASSWORD=$("$(dirname "$0")/../bin/get-django-setting" LOCAL_DATABASE_PASSWORD)
     DBNAME=zulip
-    SEARCH_PATH="$USERNAME",public
+    SEARCH_PATH="$USERNAME",public,pgroonga,pg_catalog
```

Comment:
PostgreSQL searches `pg_catalog` first if it's not specified in `search_path`.

http://www.postgresql.org/docs/9.5/static/runtime-config-client.html :

> The system catalog schema, pg_catalog, is always searched, whether it is mentioned in the path or not. If it is mentioned in the path then it will be searched in the specified order. If pg_catalog is not in the path then it will be searched before searching any of the path items.

`pgroonga` schema provides `@@` operator. It's also defined in `pg_catalog`. If we just specify only `pgroonga`, `@@` operator is never used. Because `pg_catalog` is searched first.

It causes a problem when sequential scan is used by PostgreSQL. Both of `@@` operator in `pg_catalog` and `@@` operator in `pgroonga` provide full-text search feature. But `@@` operator in `pg_catalog` doesn't support languages that doesn't put space between words such as Japanese and Chinese. In sequential scan, we need to use `@@` operator in `pgroonga` to support all languages.